src/components/general/header/components/Login.tsx

```diff
@@ -2,18 +2,17 @@
 import KeycloakService from 'keycloak/Keycloak';
 import { Row, Col } from 'react-bootstrap';
```

Comment:
I see the mix between single and double quotes, not sure if it needs fixing, maybe not in this PR.

Reply:
My intent is to use single quotes with import statements, will replace any old imports with double quotes I encounter